panel/config/configpanelwidget.cpp

```diff
@@ -305,6 +305,7 @@ int ConfigPanelWidget::indexForPosition(int screen, ILXQtPanel::Position positio
 ConfigPanelWidget::~ConfigPanelWidget()
 {
     delete ui;
+    ui = nullptr;
```

Comment:
..still the same pattern as noted before.... and the same applies for next occurrences.

Reply:
Oh, I forgot to remove that